src/components/Info/Form.js

```diff
@@ -1,15 +1,30 @@
-import React, { useState } from 'react'
-import { Button, Form } from './elements'
+import React from 'react'
+import { useContext } from 'react'
```

Comment:
The imports from react should be consolidated into one line like:
```suggestion
import React, { useContext, useReducer } from 'react';
```